### PR TITLE
Release v0.22.0

### PR DESCRIPTION
**Pull request by vincethecoder**
Tertiary visual instructions and lane indications changes: https://github.com/mapbox/MapboxDirections.swift/compare/v0.21.0...master
/cc @mapbox/navigation-ios

### CODE REVIEW

File: CHANGELOG.MD [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Changes to the Mapbox Directions SDK for iOS
 
-## master
+## v0.22.0
 
 * Removed `MBAttributeOpenStreetMapNodeIdentifier, as it is no longer being tracked by the API. This is a breaking change.
-* Fixed a bug which caused coordinates to be off by a factor of 10 when requesting `.polyline6` shape format.
\ No newline at end of file
+* Fixed a bug which caused coordinates to be off by a factor of 10 when requesting `.polyline6` shape format. []()
```
Comment (bsudekum) [inline CHANGELOG.MD]: Add an actual link here.

File: CHANGELOG.MD [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # Changes to the Mapbox Directions SDK for iOS
 
-## master
+## v0.22.0
 
 * Removed `MBAttributeOpenStreetMapNodeIdentifier, as it is no longer being tracked by the API. This is a breaking change.
```
Comment (bsudekum) [inline CHANGELOG.MD]: Add a link here too to the PR.

**Comment by bsudekum** [comment]
@vincethecoder the podspec still needs to be updated.

**Comment by vincethecoder** [comment]
@bsudekum done ✅

**Pull request merged**